mangaki/mangaki/settings.py

```diff
@@ -119,6 +121,28 @@
     }
 ]
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'level': 'INFO',
+            'class': 'logging.StreamHandler',
+        },
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': 'debug.log'
+        }
+    },
+    'loggers': {
+        'mangaki': {
+            'handlers': ['console', 'file'],
+            'level': 'DEBUG'
```

Comment:
Question: Do we want to write on `debug.log` during production?
Should the level here `INFO` by default, and `DEBUG` when in Debug mode?